apps/axistream/src/hls_int.hh

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#pragma GCC system_header
+#include "ap_int.h"
```

Comment:
why?

Reply:
Just to get rid of warnings. Otherwise, compiling with -Wall -Werror doesn't work.

Reply:
I think it was more than that - like signed/unsigned compare and maybe some others. I'll double-check though.

Reply:
The intention is that those classes are otherwise warning clean.  if you can point me to the problematic bits, I'll point the development team at them.

Reply:
Here are the errors I get if I remove these `system_header` pragmas: https://gist.github.com/orangeturtle739/ea24d2d937ba37300f5607502f9962cc.